Added weightage rebalancing into JLP guide (#465)
jup-ag/space-station: guides/7-jlp/2-How-JLP-Works.md

The swap fee for the pool typically ranges between 0% and 2%.

### Pool Weightage

> In JLP, tokens are assigned a **target weight**, which is set by the team. 

Reviewer: ```suggestion
In the JLP pool, each asset is assigned a **target weight**, which is set by the team. 
```

Non-blocking; I feel this reads better, WDYT?

![jlp-weightage](jlp-weightage.png)

Experts like [Gauntlet](https://www.gauntlet.xyz/) and [Chaos Labs](https://chaoslabs.xyz/) conduct regular risk assessments and market analyses in response to the evolving market conditions, and discuss with the community before presenting to the team for rollout. 

:::tip
 The risk assessments can be accessed via **[Jupiter's research forum](https://www.jupresear.ch/tag/risk)**.
:::

### Maintaining Target Weight

> The weightages of each asset in JLP will differ based on market activity, particularly Spot Trading or Deposit & Withdrawal of assets into JLP.

Reviewer: ```suggestion
The weightages of each asset in the JLP pool will differ based on market activity, particularly Spot Trading or Deposit & Withdrawal of assets into JLP.
```

Non-blocking; I feel it'd be good to refer to the liquidity pool as the `JLP pool`, because I think `JLP` itself refers to the token. WDYT?

To help maintain the Target Weight, JLP dynamically sets a Swap Fee or Mint/Redeem Fee.

> ![weightage-rebalancing](weightage-rebalancing.jpg)

Reviewer: Non-blocking; this is not necessary, but WDYT of adding a visual for the other side as well when the current weightage is less than the target weightage?

Author: Good suggestion - i'll add both sides into the graphic

> Simply put: Transactions that shift an asset’s current weightage **further away** from the target weightage incur additional fees while transactions that shift it **closer to** the target weightage will incur close to 0 fees. This is based on the fee incurred when minting or burning JLP during the swap. 

Reviewer: I think a more accurate way of saying this would be:

```suggestion
Simply put: Transactions that shift an asset’s current weightage **further away** from the target weightage incur additional fees while transactions that shift it **closer to** the target weightage will receive a discount on fees. This is based on the fee incurred when minting or burning JLP during the swap. 
```

WDYT?

> This allows JLP to maintain its target weight amid the high volume of spot trading or deposit/withdrawal activity.

Reviewer: ```suggestion
This allows JLP to maintain its target weight as liquidity providers are incentivized to maintain the target weightage amid high volumes of spot trading or deposit/withdrawal activity.
```

Non-blocking; I think we can mention that the fee tax / rebate mechanism encourages traders to maintain the target weightage as well, WDYT of this?
